lessons/pydata/pandas/index.ipynb

```diff
@@ -0,0 +1,2872 @@
+       "0      False\n",
+       "1      False\n",
+       "2       True\n",
+       "3       True\n",
+       "4       True\n",
+       "       ...  \n",
+       "802     True\n",
+       "803     True\n",
+       "804    False\n",
+       "805     True\n",
+       "806     True\n",
+       "Name: speed, Length: 807, dtype: bool"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.speed >= 65"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I další známé operátory lze použít stejným způsobem a aplikovat je na sloupec a jednu hodnotu či na více sloupců najednou."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0       4.5\n",
+       "1       6.0\n",
+       "2       8.0\n",
+       "3       6.5\n",
+       "4       8.0\n",
+       "       ... \n",
+       "802     7.3\n",
+       "803    12.1\n",
+       "804     1.3\n",
+       "805    10.7\n",
+       "806    14.3\n",
+       "Name: speed, Length: 807, dtype: float64"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data[\"speed\"] / 10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0       2205\n",
+       "1       3780\n",
+       "2       6640\n",
+       "3       1677\n",
+       "4       3364\n",
+       "       ...  \n",
+       "802     4489\n",
+       "803     5329\n",
+       "804    12871\n",
+       "805     2809\n",
+       "806     6600\n",
+       "Length: 807, dtype: int64"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.hp * data.defense"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pandas obsahuje i řadu užitečných metod pro práci s řetězci."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0       GREEN\n",
+       "1       GREEN\n",
+       "2       GREEN\n",
+       "3         RED\n",
+       "4         RED\n",
+       "        ...  \n",
+       "802    PURPLE\n",
+       "803    PURPLE\n",
+       "804      GRAY\n",
+       "805     WHITE\n",
+       "806    YELLOW\n",
+       "Name: color, Length: 807, dtype: object"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.color.str.upper()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Výsledek takové operace lze velmi snadno přidat zpět k původní tabulce jako nový sloupec."
```

Comment:
Co to trochu ještě uvést že "tabulka" (`DataFrame`) lze měnít - např. přidat sloupec. A přidat něco jednoduššího než `data.color.str.upper()`, třeba `data["fast"] = data.speed >= 65`